src/torchjd/autojac/_utils.py

```diff
@@ -30,3 +30,30 @@ def _check_retain_graph_compatible_with_chunk_size(
             "When using `retain_graph=False`, parameter `parallel_chunk_size` must be `None` or "
             "large enough to compute all gradients in parallel."
         )
+
+
+def _get_leafs_of_autograd_graph(roots: list[Tensor], excluded: set[Tensor]) -> set[Tensor]:
+    """
+    Gets the leafs of the autograd graph of all tensors in `roots`, excludes leafs that are in the
+    autograd graph of any tensor in `excluded`.
+    :param roots: Roots of the autograd graphs.
+    :param excluded: Tensors excluded from the search.
```

Comment:
If I understand correctly, we also exclude tensors that are descendants of the `excluded` tensors in the backward computation graph, right? Because of the condition in this line:
```python
node_stack += [
    child[0]
    for child in current_node.next_functions
    if child[0] is not None and child[0] not in excluded_nodes
]
```
If so, the documentation of this function and of its `excluded` param is not accurate enough.